api/billingd/service/service.go

```diff
@@ -1019,14 +1018,34 @@ func (s *Service) reportUsage() error {
 }
 
 func (s *Service) reportCustomerUsage(ctx context.Context, cus *Customer) error {
+	summary := map[string]interface{}{
+		"grace_period_start":   s.analytics.FormatUnix(cus.GracePeriodStart),
+		"invoice_period_end":   s.analytics.FormatUnix(cus.InvoicePeriod.UnixEnd),
+		"invoice_period_start": s.analytics.FormatUnix(cus.InvoicePeriod.UnixStart),
+		"subscription_status":  cus.SubscriptionStatus,
+		"billable":             cus.Billable,
+		"delinquent":           cus.Delinquent,
+		"account_status":       cus.AccountStatus(),
+	}
+	deps, err := s.cdb.CountDocuments(ctx, bson.M{"parent_key": cus.Key})
+	if err == nil {
+		summary["dependents"] = deps
```

Comment:
let it fail silently and grab it on the daily cron

Reply:
If there's an error there, would be bad, probably worth returning as normal. If there are no results, `deps` will be 0.

Reply:
kk, will do